jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/java/AbtractJavaContext.java

```diff
@@ -22,29 +21,27 @@
 import org.eclipse.jdt.core.ITypeRoot;
 import org.eclipse.jdt.core.dom.CompilationUnit;
 import org.eclipse.jdt.internal.core.manipulation.dom.ASTResolving;
-
-import org.eclipse.lsp4jakarta.jdt.core.JDTUtils;
+import org.eclipse.lsp4jakarta.jdt.core.utils.IJDTUtils;
 
 /**
- * Abstract class for Java context for a given compilation unit. Reused from
- * https://github.com/eclipse/lsp4mp/blob/master/microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/java/AbtractJavaContext.java
+ * Abstract class for Java context for a given compilation unit.
  *
- * @author credit to Angelo ZERR
+ * @author Angelo ZERR
  *
  */
-public abstract class AbstractJavaContext {
+public abstract class AbtractJavaContext {
```

Comment:
I'm guessing this came from LSP4MP, but wanted to point out the typo (abtract vs. abstract) in the name of this class.

Reply:
Thanks for catching this Michael. I will update it.